griptape/tasks/code_execution_task.py

```diff
@@ -1,13 +1,13 @@
 from __future__ import annotations
-from attr import define, field
+from attr import define, field, Factory
 from griptape.artifacts import BaseArtifact, ErrorArtifact
 from griptape.tasks import BaseTextInputTask
 from typing import Callable
 
 
 @define
 class CodeExecutionTask(BaseTextInputTask):
-    run_fn: Callable[[CodeExecutionTask], BaseArtifact] = field(kw_only=True)
+    run_fn: Callable[[CodeExecutionTask], BaseArtifact] = field(default=Factory(lambda: lambda task: task.input))
```

Comment:
Rather than using `CodeExecutionTask` as an "identity" `Task`, how about creating one explicitly for this purpose?

I'd prefer readability over re-usability for this use case, particularly since the amount of code reuse is trivial. I'd call the new class something like `PassTask` or `NoOpTask` or `InputTask` or `IdentityTask`. (`IdentityTask` is my favorite, but I don't think it'll be everyone elses)

Reply:
I had originally created a `NoOpTask` but it felt out of scope for this. Now that you bring it up maybe that is the better solution here... I'll run with that.